GitUI/UserControls/RevisionGrid/RevisionGridControl.cs

```diff
@@ -402,21 +402,21 @@ public void SetAndApplyPathFilter(string filter)
             PerformRefreshRevisions();
         }
 
-        private void InitiateRefAction(IReadOnlyList<IGitRef>? refs, Action<IGitRef> action, FormQuickGitRefSelector.Action actionLabel)
+        private void InitiateRefAction(IReadOnlyList<IGitRef>? getRefs, Action<IGitRef> action, FormQuickGitRefSelector.Action actionLabel)
```

Comment:
```suggestion
        private void InitiateRefAction(IReadOnlyList<IGitRef>? refs, Action<IGitRef> action, FormQuickGitRefSelector.Action actionLabel)
```

Reply:
Yes reverting this.

Plan to merge later today.